platforms/ios/lib/WysiwygComposer/Sources/WysiwygComposer/Extensions/UITextView.swift

```diff
@@ -33,8 +33,15 @@ extension UITextView {
             // avoids an issue with autocapitalization.
             self.selectedRange = .zero
             self.selectedRange = content.selection
+
+            self.drawBackgroundStyleLayers()
         }
     }
+
+    override open func draw(_ rect: CGRect) {
+        super.draw(rect)
+        drawBackgroundStyleLayers()
+    }
```

Comment:
I would move this draw override to our custom `PlaceholdableTextView` and maybe change its name to WysiwygTextView or something like that. Extending the textview openly here in a library could create undesired or unneeded effects on the apps that are going to use it.

Reply:
I would still make `drawBackgroundStyleLayers()` public, so that we can still use this (and we may want it for the timeline on element-ios maybe?) to use the custom background style inside only the specific text views that we want, by overriding only those.